Basic-Car-Maintenance/Documentation.docc/getting-started.tutorial

```diff
@@ -108,4 +108,44 @@
             }
         }
     }
+
+    @Section(title: "Setting Up Firebase Local Emulator") {
```

Comment:
this new section should actually be before the "Starting on an issue" section as it is in the CONTRIBUTING file